aztec/src/main/res/values/strings.xml

```diff
@@ -2,7 +2,7 @@
 
     <!-- GENERAL -->
     <string name="cursor_moved">Cursor moved</string>
-    <string name="samsung_disabled_custom_clipboard">Sorry, this feature is disabled on Android 8. Please use the Paste action instead.</string>
+    <string name="samsung_disabled_custom_clipboard">Sorry, this feature is disabled on Android %1$s. Please use the Paste action instead.</string>
```

Comment:
@aforcier I'm not 100% sure so I'll better ask. Consider the following scenario
1. `samsung_disabled_custom_clipboard` is created
2. It gets translated into Spanish
3. It gets released
4. We update the string in English, but we don't translated it yet
5. It gets released
6. What version of the string will be used on a Spanish device?

I'm asking, because if we use version from step `2.` the app will crash on `context.getString(R.string.samsung_disabled_custom_clipboard, Build.VERSION.RELEASE)` as the original string doesn't contain any placeholder.

Reply:
` If there are more arguments than format specifiers, extra arguments are ignored.`
So it seems the app won't crash after all, but I'm curious how the translations work anyway:).

Reply:
You know, I'm not sure 🤔 it should depend on whether the GlotPress job updates translation files or completely replaces them each time, I think I once knew but forgot. @loremattei do you know the answer to this one? 🙂

Reply:
Sure! So, when the new English string is uploaded to GlotPress during code freeze, the translation is invalidated. When we download the strings two weeks later to build the final binary, if the updated string has not not been translated yet, it's removed from the Spanish (or whatever) strings.xml and the app will show the English version. 

It works that the translated files are completely replaced by the newest ones, so untranslated strings are not in them.

Reply:
Thank you for the clarification Lorenzo! ;)